src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1587,16 +1590,16 @@ private static VersionInfo GetVersionInfo()
         }
 
         // Epsilon for 0-comparisons
-        private const Float Epsilon = (Float)1e-15;
-        private const Float MinToReturn = Epsilon; // max predicted is 1 - min;
-        private const Float MaxToReturn = 1 - Epsilon; // max predicted is 1 - min;
+        private const float Epsilon = (float)1e-15;
+        private const float MinToReturn = Epsilon; // max predicted is 1 - min;
+        private const float MaxToReturn = 1 - Epsilon; // max predicted is 1 - min;
 
         private readonly IHost _host;
-        private readonly Float[] _mins;
-        private readonly Float[] _maxes;
-        private readonly Float[] _values;
+        private readonly float[] _mins;
+        private readonly float[] _maxes;
+        private readonly float[] _values;
 
-        internal PavCalibrator(IHostEnvironment env, Float[] mins, Float[] maxes, Float[] values)
+        internal PavCalibrator(IHostEnvironment env, float[] mins, float[] maxes, float[] values)
```

Comment:
>internal PavCalibrator(IHostEnvironment env, float[] mins, float[] maxes, float[] values) [](start = 8, length = 89)

All other calibrators has exposed public constructor and public properties.
Here you have internal constructor and private readonly  _mins, _maxes, _values.